function/pkg/operator/triggers_operator.go

```diff
@@ -4,22 +4,50 @@ import (
 	"context"
 
 	"github.com/kyma-incubator/hydroform/function/pkg/client"
+	"github.com/kyma-incubator/hydroform/function/pkg/resources/types"
 	"github.com/pkg/errors"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
 )
 
 const message = "ownerID"
 
+type FnRef struct {
+	name      string
+	namespace string
+}
+
 type triggersOperator struct {
+	fnRef FnRef
 	items []unstructured.Unstructured
 	client.Client
 }
 
-func NewTriggersOperator(c client.Client, u ...unstructured.Unstructured) Operator {
+func NewTriggersOperator(c client.Client, fnName, fnNamespace string, u ...unstructured.Unstructured) Operator {
 	return &triggersOperator{
 		Client: c,
 		items:  u,
+		fnRef: FnRef{
+			name:      fnName,
+			namespace: fnNamespace,
+		},
+	}
+}
+
+func buildPredicate(fnRef FnRef, items []unstructured.Unstructured) func(map[string]interface{}) (bool, error) {
```

Comment:
I think it's worth to add some short comment describing how this predicate behaves.